### ISSUE TITLE
Remove external assets, ship them with materialious on same domain

### ISSUE THREAD

**Issue by Sommerwiesel** [WardPearce/Materialious#173]
**Describe the bug**
All loading of external assets (currently only fonts) should be removed.
These assets should ship with materialious and be loaded by materialious from the same domain.

Background: A lot of Invidious users have very strict browser settings. I personally use arkenfox with uBlock Origin and have disabled loading of all external fonts.

On your and my instance, this results in all symbols originating from [material-symbols-outlined.woff2](https://cdn.jsdelivr.net/npm/beercss@3.5.1/dist/cdn/material-symbols-outlined.woff2) simply don't load until I have whitelisted the page.

**To Reproduce**
1. Have a strict browser setting (i.e. arkenfox) or uBlock Origin with blocking external ressources or plugins like Decentralize / Privacy Possum. Note: A lot of Invidious users will have something like that.
1. Go to '[materialious](https://materialio.us/)'
3. Have no fonts load at all:
`downloadable font: download failed (font-family: "Material Symbols Outlined" style:normal weight:400 stretch:100 src index:1): bad URI or cross-site access not allowed source: https://cdn.jsdelivr.net/npm/beercss@3.5.1/dist/cdn/material-symbols-outlined.woff2`

**Expected behavior**

Materialious should ship with all assets and don't load anything from an external source.

**Screenshots**
[materialious1.png](https://postimg.cc/m1Wqrft4)

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: Firefox ESR (arkenfox.js)
 - Version 115.10.0ESR

**Additional context**
Materialious seems to ship with the fonts file already but for some reason it is not loaded correctly. This bug should therefore probably be a really quick fix.

**Comment by WardPearce** [comment]
Good catch, this seems like a upstream issue with BeerCSS. Not sure why they decided to load fonts like this, as I agree typically isn't best practice to do so.

**Comment by leonardorafael** [comment]
@WardPearce nice project!

https://github.com/beercss/beercss/blob/main/src/cdn/settings/fonts.css

As you can see, the external url is just a font fallback when the local font fails (it's highly recommended to have a font fallback). The modern bundlers should put the local fonts on the right path automatically.

Just verify if the bundler is copying the fonts on the right path. If not, do that mannually.

**Comment by WardPearce** [comment]
Okay thanks, I'll check it out. Also I love beercss!!!

**Comment by leonardorafael** [comment]
@WardPearce, seems the fonts are loading correctly here https://materialio.us/

![image](https://github.com/beercss/beercss/assets/12979016/c2aac929-78c8-4df9-ac52-ff7431827bb8)

**Comment by WardPearce** [comment]
> @WardPearce, seems the fonts are loading correctly here https://materialio.us/
> 
> ![image](https://private-user-images.githubusercontent.com/12979016/326434191-c2aac929-78c8-4df9-ac52-ff7431827bb8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQzOTM5NjQsIm5iZiI6MTcxNDM5MzY2NCwicGF0aCI6Ii8xMjk3OTAxNi8zMjY0MzQxOTEtYzJhYWM5MjktNzhjOC00ZGY5LWFjNTItZmY3NDMxODI3YmI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI5VDEyMjc0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUwZDY0ZWY1ZDY5OGE3N2Q1OTg4Mjg0YTY2YWEwODkxZjAwNDY4YTBjMzhlMDkxMzIxZjg5NGVkMDdjZjY3YTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3KQwXM60YQnbV3zZ8CW4hJQyN-WNdS2T-UbpIClIZMQ)

Yea odd, seems to be correct for me too.

**Comment by WardPearce** [comment]
> @WardPearce Additional information:
> 
> For some reason, uBlock Origin blocks "https://materialio.us/_app/immutable/assets/material-symbols-outlined.CFhFrqCB.woff2" as part of the setting "block all external fonts".

Aah maybe block all externals fonts literally blocks all fonts what aren't system default

**Comment by WardPearce** [comment]
> > > @WardPearce Additional information:
> > > For some reason, uBlock Origin blocks "https://materialio.us/_app/immutable/assets/material-symbols-outlined.CFhFrqCB.woff2" as part of the setting "block all external fonts".
> > 
> > 
> > Aah maybe block all externals fonts literally blocks all fonts what aren't system default
> 
> No, the custom font and bootstrap icon font that I use on my homepage both work perfectly fine.

odd, extra odd
